pom.xml

```diff
@@ -604,6 +604,10 @@
                                 <configuration>
                                     <!--suppress UnresolvedMavenProperty -->
                                     <keyname>${releaseSigningKey}</keyname>
+                                    <gpgArguments>
```

Comment:
Without this, gpg won't read the passphrase from the environment.   Even with this, interactive use works fine (at least on a Mac).